src/components/Verse/VerseText.module.scss

```diff
@@ -40,8 +40,20 @@
   flex-wrap: wrap;
 }
 
-@include utility.generate-font-sizes();
+@include utility.generate-font-sizes("code_v1");
+@include utility.generate-font-sizes("code_v2");
+@include utility.generate-font-sizes("text_indopak");
+@include utility.generate-font-sizes("qpc_uthmani_hafs");
 
 .highlighted {
   background-color: var(--color-background-alternative-faded);
 }
+
+// make the text slightly bigger on translation or tafsir mode
+.tafsirOrTranslationMode {
```

Comment:
So the arabic font size in translation mode is basically 1.2x what's on the reading mode?

Reply:
correct, on mobile.